src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -174,6 +174,17 @@ void maybeInitializePreprocessorSymbolTable(AbstractCompiler compiler) {
       }
     }
   }
+  
+  @Override
+  protected List<PassFactory> getAlwaysPreChecks() {
+    List<PassFactory> preChecks = new ArrayList<>();
+    
+    if (options.wrapGoogModulesForWhitespaceOnly) {
```

Comment:
@blickly I pushed a change that simplifies things somewhat. Please see TODO added to Compiler change.